adoc/scripts/reflow.py

```diff
@@ -53,7 +53,7 @@
 # A single letter followed by a period, typically a middle initial.
 endInitial = re.compile(r'^[A-Z]\.$')
 # An abbreviation, which does not (usually) end a line.
-endAbbrev = re.compile(r'(e\.g|i\.e|c\.f|\bvs\b|\bco\b|\bltd\b|\bch\b)\.$', re.IGNORECASE)
+endAbbrev = re.compile(r'(e\.g|i\.e|c\.f|\bvs\b|\bco\b|\bltd\b|\bch\b|\bcf\b)\.$', re.IGNORECASE)
```

Comment:
We already have "c.f." in this list, but it has a period between the "c" and "f".  I think your spelling is the correct one, so can you remove the old "c.f"?  I did a quick search of the spec, and I did not see any occurrences of "c.f.".

Reply:
:facepalm: I double-checked the abbreviations earlier and still missed it. I've removed the version with the extra `.`.